stable/xray/templates/xray-statefulset.yaml

```diff
@@ -120,6 +120,15 @@ spec:
       - name: {{ .Values.router.name }}
         image: '{{ .Values.router.image.repository }}:{{ default .Chart.AppVersion .Values.router.image.version }}'
         imagePullPolicy: {{ .Values.router.image.imagePullPolicy }}
+        command:
+          - '/bin/sh'
+          - '-c'
+          - >
+          {{- with .Values.common.preStartCommand }}
+            echo "Running custom common preStartCommand command";
+            {{ tpl . $ }};
+          {{- end }}
+            /opt/jfrog/router/app/bin/entrypoint-router.sh;
```

Comment:
Use `exec` before. Like we have in artifactory charts.

Reply:
Please see comment below. I would expect this to be the same as all the other prestartcommands but we could apply this throughout if necessary. The goal of this PR was only to make router match the other 4 containers. It seems like somebody forgot about this container to be honest.